src/listeners/command-handler/CoreCommandAccepted.ts

```diff
@@ -12,6 +12,11 @@ export class CoreListener extends Listener<typeof Events.CommandAccepted> {
 		const args = await command.preParse(message, parameters, context);
 		try {
 			message.client.emit(Events.CommandRun, message, command, { ...payload, args });
+			if (command.typing || (command.typing === undefined && command.container.client.options.typing)) {
+				message.channel.sendTyping().catch((e) => {
+					throw e;
```

Comment:
uh.. rethrowing the error? Why?

Reply:
I wasn't sure what to do with it, so my first thought was "re-throw it so it is catched later".

However, now I realize that would halt the command execution after it has been accepted... my terrible bad, I didn't think about that 😅. (Although, when does `sendTyping()` fails?).

Would you mind pointing what to do with that error? Discard it?

Thanks in advance. ^^

Reply:
Emit a `commandTypingError` event perhaps?